src/reformat_libyuv.c

```diff
@@ -220,12 +235,12 @@ avifResult avifImageRGBToYUVLibYUV8bpc(avifImage * image, const avifRGBImage * r
             typedef int (*RGBtoY)(const uint8_t *, int, uint8_t *, int, int, int);
             // Second dimension is for avifRange.
             RGBtoY lutRgbToY[AVIF_RGB_FORMAT_COUNT][2] = {
-                { NULL, NULL },             // RGB
+                { NULL, RAWToJ400 },        // RGB
```

Comment:
Nit: I don't have a preference, but if would be good to make `avifRange` either the first dimension or the last dimension in both lookup tables.

Reply:
Done at https://github.com/AOMediaCodec/libavif/pull/1082